CONTRIBUTING.md

```diff
@@ -89,24 +89,62 @@ The changelog should follow the following convention:
 
 **README**
 
-The `README.md` should cover the following territory:
+The `README.md` should follow this template:
 
 ```
+# (package name) &middot; (badges)
+
 ## Description
 (high-level summary that will be accessible to product/UX/Test, plus link to Storybook, explain the different cases)
+
 ## When to use this component
 (including where it is currently used)
-## When to not use this component
+
+## When not to use this component
 (if this is not known, can leave the section empty)
+
+## Installation
+`npm install (package-name)`
+
+## Props
+(a table of all the component's props, following the template below)
+
+| Argument  | Type                | Required | Example         |
+|-----------|---------------------|----------|-----------------|
+| prop-name | description of type | Yes/No   | `example input` |
```

Comment:
Should we have a `Default` column?

And a `Description`?

Otherwise it's just:

```
| foo | boolean | Yes  | true |
```

...and would need some context! 😄 

So I'd suggest:

```
| Argument  | Type                      | Required | Default | Description                                |
|------------|------------------|----------|---------|-------------------------------|
| foo             | boolean                 | No           | false     | true if you want to bar the baz |
```

(Probably don't need Example _and_ Description, come to think of it)

Reply:
I've added a `Default` column. I agree somewhat on the `Description`, but updating these across all READMEs has already proven a _very_ time-consuming task & I don't feel it's worth the investment -- if the purpose isn't clear from the combination of a name, type (which in some cases already has a little more info), default & example, then it's a strong sign we should be re-evaluating the props & their identifiers in the first place.